source/GameInterface/Services/MobileParties/Messages/Control/UpdateMobilePartyControl.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Common.Messaging;
+
+namespace GameInterface.Services.MobileParties.Messages.Control
+{
+    public record UpdateMobilePartyControl : ICommand
+    {
+        public string PartyId { get; }
+
+        public bool IsRevocation { get; }
```

Comment:
Add xml comment for this field. It is difficult to understand what it does from a glance